test/functional/rpc_signrawtransaction.py

```diff
@@ -334,6 +334,56 @@ def test_signing_with_cltv(self):
         assert_equal(signed["complete"], True)
         self.nodes[0].sendrawtransaction(signed["hex"])
 
+    def signrawtransactionwithwallet_tests(self):
```

Comment:
Looks like the test could use renaming In the first commit? maybe something like this
```diff
diff --git a/test/functional/rpc_signrawtransaction.py b/test/functional/rpc_signrawtransaction.py
index 29cb116793..8da2cfa72b 100755
--- a/test/functional/rpc_signrawtransaction.py
+++ b/test/functional/rpc_signrawtransaction.py
@@ -334,10 +334,10 @@ class SignRawTransactionsTest(BitcoinTestFramework):
         assert_equal(signed["complete"], True)
         self.nodes[0].sendrawtransaction(signed["hex"])
 
-    def signrawtransactionwithwallet_tests(self):
+    def test_signing_with_missing_prevtx_info(self):
         txid = "1d1d4e24ed99057e84c3f80fd8fbec79ed9e1acee37da269356ecea000000000"
         for type in ["bech32", "p2sh-segwit", "legacy"]:
-            self.log.info(f"Test signrawtransactionwithwallet with missing prevtx info ({type})")
+            self.log.info(f"Test signing with missing prevtx info ({type})")
             addr = self.nodes[0].getnewaddress("", type)
             addrinfo = self.nodes[0].getaddressinfo(addr)
             pubkey = addrinfo["scriptPubKey"]
@@ -393,7 +393,7 @@ class SignRawTransactionsTest(BitcoinTestFramework):
         self.test_fully_signed_tx()
         self.test_signing_with_csv()
         self.test_signing_with_cltv()
-        self.signrawtransactionwithwallet_tests()
+        self.test_signing_with_missing_prevtx_info()
```

```
$ test/functional/rpc_signrawtransaction.py 
2022-05-30T13:39:10.050000Z TestFramework (INFO): Initializing test directory /tmp/bitcoin_func_test_ukdy99c_
2022-05-30T13:39:10.954000Z TestFramework (INFO): Test valid raw transaction with one input
2022-05-30T13:39:10.958000Z TestFramework (INFO): Test script verification errors
2022-05-30T13:39:10.969000Z TestFramework (INFO): Test signing transaction to P2SH-P2WSH addresses without wallet
2022-05-30T13:39:12.667000Z TestFramework (INFO): Test with a P2PKH script as the witnessScript
2022-05-30T13:39:12.822000Z TestFramework (INFO): Test with a P2PK script as the witnessScript
2022-05-30T13:39:12.970000Z TestFramework (INFO): Test OP_1NEGATE (0x4f) satisfies BIP62 minimal push standardness rule
2022-05-30T13:39:12.972000Z TestFramework (INFO): Test correct error reporting when trying to sign a locked output
2022-05-30T13:39:13.519000Z TestFramework (INFO): Test signing a fully signed transaction does nothing
2022-05-30T13:39:14.504000Z TestFramework (INFO): Test signing a transaction containing a fully signed CSV input
2022-05-30T13:39:15.920000Z TestFramework (INFO): Test signing a transaction containing a fully signed CLTV input
2022-05-30T13:39:16.368000Z TestFramework (INFO): Test signing with missing prevtx info (bech32)
2022-05-30T13:39:16.458000Z TestFramework (INFO): Test signing with missing prevtx info (p2sh-segwit)
2022-05-30T13:39:16.558000Z TestFramework (INFO): Test signing with missing prevtx info (legacy)
2022-05-30T13:39:16.699000Z TestFramework (INFO): Stopping nodes
2022-05-30T13:39:16.904000Z TestFramework (INFO): Cleaning up /tmp/bitcoin_func_test_ukdy99c_ on exit
2022-05-30T13:39:16.904000Z TestFramework (INFO): Tests successful
```

Reply:
Also in the first commit
```diff
diff --git a/test/functional/rpc_rawtransaction.py b/test/functional/rpc_rawtransaction.py
index fecb8310b9..54357ccd31 100755
--- a/test/functional/rpc_rawtransaction.py
+++ b/test/functional/rpc_rawtransaction.py
@@ -7,7 +7,6 @@
 Test the following RPCs:
    - getrawtransaction
    - createrawtransaction
-   - signrawtransactionwithwallet
    - sendrawtransaction
    - decoderawtransaction
 """
```

Reply:
> Edit: hm, there are still a few in raw_multisig_transaction_legacy_tests() so perhaps never mind

Yeah, I'd say it's better to leave it :)